portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/pgstaging/PGStagingWithVersioning.testcase

```diff
@@ -189,9 +189,9 @@ definition {
 				key_tooltipInfo = "Page variations allow you to work in parallel in different versions of the same page inside this pages variation. When publishing to live, the version marked as ready to publish is the one that is processed.",
 				locator1 = "Staging#STAGING_TOOLTIP");
 
-			AssertVisible(
-				key_content = "Page variations allow you to work in parallel in different versions of the same page inside this pages variation. When publishing to live, the version marked as ready to publish is the one that is processed.",
-				locator1 = "Tooltip#FLOATING_TOOLTIP_CONTENT");
+			AssertTextPresent(
```

Comment:
Please reverse-check the Tooltip#FLOATING_TOOLTIP_CONTENT locator and see if we should update/extend this locator. If updating or extending this locator would cause other failures, please add a new locator for the current one: **//div[contains(@class,'tooltip-inner')]**. The main goal is that reuse locators as much as possible.